packages/tests/server/react/polymorphism.legacy.factory.tsx

```diff
@@ -0,0 +1,106 @@
+//
```

Comment:
Ignore these `.legacy` files, they're just straight renames of the old tests so we can review the diff of the new ones